common-service-core/build.gradle

```diff
@@ -9,7 +9,11 @@ dependencies {
 
     compileOnly(libraries.javax_servlet_api)
     compileOnly(libraries.lombok)
+    compileOnly(libraries.spring_boot_starter_cache)
```

Comment:
Our intention was to keep ```common-service-core``` as isolated from spring dependencies.  I would like to ask that why don't you keep your classes( which depend on these spring-deps) in ```apiml-common``` module?

Reply:
Spring libraries are needed only during compilation and usage of cache class.